recipes-browser/wpewebkit/wpewebkit_2.46.1.bb

```diff
@@ -0,0 +1,15 @@
+require wpewebkit.inc
+require conf/include/devupstream.inc
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+
+SRC_URI = "https://wpewebkit.org/releases/${BPN}-${PV}.tar.xz;name=tarball"
+
+SRC_URI[tarball.sha256sum] = "1e0aaf870f36001c42b1ce5a2027b4101bed878746e437cc6d6fed0693afe9ad"
+
+SRC_URI:class-devupstream = "git://github.com/WebKit/WebKit.git;protocol=https;branch=main"
+
+SRCREV:class-devupstream = "8db1e051cf4a6d7877dd51b81b478e8367d3f052"
```

Comment:
eacf9977760d4ea5dfcb736be73ae48c7df16f27  ？

Reply:
that tags belongs to the wpewebkit-2.46.1 branch. Notice that the SRC_URI:class-devupstream points to the `main` branch what is the default upstream branch for development. In general, I use the latest common commit between the release branch and main (`git merge-base main wpewebkit-2.46.1`) to set this value.

Reply:
@Qingwu-Li @aperezdc Ok, your argument has changed my point of view. I have added this additional commit: https://github.com/Igalia/meta-webkit/pull/509/commits/abd8ebd4425ed48c0b6b91a700c3a7f1e01d6e42, as we were using devupstream differently for referencing the upstream top-of-tree